.bluemix/toolchain.yml

```diff
@@ -122,6 +122,12 @@ services:
             } else {
               'gitlab-pr-listener';
             }
+          IBMCLOUD_API: >
+            if ( $env.env_id === 'ibm:ys1:us-south') {
```

Comment:
Should probably support `ibm:ys1:eu-gb` as well.

Reply:
Good point. @maire-kehoe do we need to support all regions here?

Reply:
When I try to look at the toolchains in the `ibm:ys1:eu-gb` region, it shows me the same toolchain list that's in `ibm:ys1:us-south`, as in, it seems the staging London region is aliased to the same as the staging Dallas region for toolchain creation, and is not a real toolchains region. You can add support for it if you want, but I don't think that people test toolchains with that region very often.
Attempting to add a contains clause won't work as the `$env` expressions don't allow function calls.

Reply:
Now that I think about it, shouldn't the API endpoint be predicated on the `registry-region` and `prod-region` values instead? After all the API you are targeting is affected by the region of the action you are trying to do (containerize or deploy to IKS).

Reply:
The code in question is doing an ibmcloud login to get a bearer token, using the bearer token to talk to the github-consolidated-broker for the current repo tool instance, to get the git token, to talk to the git server. So it needs to login using the same iam domain as the toolchain (ys1 vs yp), but the bearer token is region independent / global, so doesn't need to login to a specific region.
Code:
- https://github.com/open-toolchain/tekton-catalog/blob/ea3a38ca9f935d827d3ee3bf959b885746553158/git/task-clone-repo.yaml#L153

Some later code will log in again using the prod region when attempting to interact with the cluster.

So the current code should hopefully be ok.